ci-operator/config/openshift/ols-load-generator/openshift-ols-load-generator-main.yaml

```diff
@@ -15,98 +15,148 @@ releases:
+  cron: 0 12 * * *
   steps:
     allow_skip_on_success: true
-    cluster_profile: aws-perfscale
+    cluster_profile: aws-perfscale-qe
     env:
-      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "3"
       COMPUTE_NODE_TYPE: m6i.xlarge
+      ES_BENCHMARK_INDEX: ols-load-test-results*
+      ES_METADATA_INDEX: perf_scale_ci*
+      ES_TYPE: qe
+      HUNTER_ANALYZE: "true"
+      LOOKBACK: "180"
+      LOOKBACK_SIZE: "15"
       OLS_TEST_DURATIONS: 5m,10m,20m
       OLS_TEST_WORKERS: "50"
       OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.xlarge
+      ORION_CONFIG: examples/ols-load-generator.yaml
+      ORION_ENVS: ols_test_workers=50
+      RUN_ORION: "true"
+      VERSION: "4.15"
       ZONES_COUNT: "3"
     test:
-    - ref: ols-load-generator-tests
+    - ref: openshift-ols-load-generator-tests
+    - ref: openshift-qe-orion
     workflow: openshift-qe-installer-aws
-  timeout: 8h0m0s
 - as: ols-load-test-100workers
-  cron: 0 18 1-7,15-21 * 1
+  cron: 0 18 * * *
   steps:
     allow_skip_on_success: true
-    cluster_profile: aws-perfscale
+    cluster_profile: aws-perfscale-qe
     env:
-      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "3"
       COMPUTE_NODE_TYPE: m6i.xlarge
+      ES_BENCHMARK_INDEX: ols-load-test-results*
+      ES_METADATA_INDEX: perf_scale_ci*
+      ES_TYPE: qe
+      HUNTER_ANALYZE: "true"
+      LOOKBACK: "180"
+      LOOKBACK_SIZE: "15"
       OLS_TEST_DURATIONS: 5m,10m,20m
       OLS_TEST_WORKERS: "100"
       OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.xlarge
+      ORION_CONFIG: examples/ols-load-generator.yaml
+      ORION_ENVS: ols_test_workers=100
+      RUN_ORION: "true"
+      VERSION: "4.15"
       ZONES_COUNT: "3"
     test:
-    - ref: ols-load-generator-tests
+    - ref: openshift-ols-load-generator-tests
+    - ref: openshift-qe-orion
     workflow: openshift-qe-installer-aws
-  timeout: 8h0m0s
 - always_run: false
   as: ols-load-test-1000workers
   steps:
     allow_skip_on_success: true
-    cluster_profile: aws-perfscale
+    cluster_profile: aws-perfscale-qe
     env:
-      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "3"
       COMPUTE_NODE_TYPE: m6i.xlarge
+      ES_BENCHMARK_INDEX: ols-load-test-results*
+      ES_METADATA_INDEX: perf_scale_ci*
+      ES_TYPE: qe
+      HUNTER_ANALYZE: "true"
+      LOOKBACK: "180"
+      LOOKBACK_SIZE: "15"
       OLS_TEST_DURATIONS: 5m
       OLS_TEST_WORKERS: "1000"
       OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.xlarge
+      ORION_CONFIG: examples/ols-load-generator.yaml
+      ORION_ENVS: ols_test_workers=1000
+      RUN_ORION: "true"
+      VERSION: "4.15"
       ZONES_COUNT: "3"
     test:
-    - ref: ols-load-generator-tests
+    - ref: openshift-ols-load-generator-tests
+    - ref: openshift-qe-orion
```

Comment:
@vishnuchalla Do we have a doc on what each test is doing ?

Reply:
Its not something different from our test plan and our automation planning.
* Test plan: https://docs.google.com/document/d/11r-6vPT-Z9nA8mMe6fa4US3OgAEUdzEuVAuY5v-wW1w/edit?tab=t.0#heading=h.66y4kqbj468a
* Automation plan: https://docs.google.com/presentation/d/148X9WQUmeuwE_T8WF-cPSi0Hq4dn49WK8LovOp2s9hY/edit#slide=id.g30b14cf01b8_1_1